yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnAllocatorSuite.scala

```diff
@@ -87,7 +87,7 @@ class YarnAllocatorSuite extends SparkFunSuite with Matchers with BeforeAndAfter
   }
 
   class MockSplitInfo(host: String) extends SplitInfo(null, host, null, 1, null) {
-    override def hashCode(): Int = Random.nextInt()
+    override def hashCode(): Int = 0
```

Comment:
Since you need to re-run the tests anyway -- also remove the unneeded import of scala.util.Random now